spec/Section 6 -- Execution.md

```diff
@@ -740,7 +740,7 @@ ResolveAbstractType(abstractType, objectValue):
 
 **Merging Selection Sets**
 
-When more than one field of the same name is executed in parallel, their
+When multiple fields with the same name or alias are executed, their
```

Comment:
"multiple" is less precise; for example:

https://www.dictionary.com/browse/multiple :

> consisting of, having, or involving **several** or **many** individuals, parts, elements, relations, etc.; manifold.

https://www.dictionary.com/browse/several :

> being **more than two** but fewer than many in number or kind:

https://www.dictionary.com/browse/many :

> constituting or forming **a large number**; numerous:

In the above definitions, "multiple" can be construed to mean any number 3 or larger. I, personally, view "multiple" as meaning 2 or more (but very strange to be used with 2); however in a spec we must be precise and not chose ambiguous language.

Please revert this change.

Reply:
The main point here is removing 'in PARALLEL', I can rephrase the 'multiple' part no problem

Reply:
also, instead of bringing in the alias ... we should just refer to `response name` as we do in the field merging algorithm.

Reply:
https://spec.graphql.org/October2021/#sec-Field-Selection-Merging

Reply:
but is that even correct? in the case of objects this is not correct since we cannot merge two object that are different